domain/src/main/java/org/oppia/android/domain/oppialogger/analytics/PerformanceMetricsLogger.kt

```diff
@@ -0,0 +1,260 @@
+package org.oppia.android.domain.oppialogger.analytics
+
+import org.oppia.android.app.model.OppiaMetricLog
+import org.oppia.android.domain.oppialogger.ApplicationStartupListener
+import org.oppia.android.util.logging.performancemetrics.PerformanceMetricsUtils
+import org.oppia.android.util.system.OppiaClock
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class PerformanceMetricsLogger @Inject constructor(
+  private val performanceMetricsController: PerformanceMetricsController,
+  private val performanceMetricsUtils: PerformanceMetricsUtils,
+  private val oppiaClock: OppiaClock
+) : ApplicationStartupListener {
+
+  /**
+   * Use a large Long value such that the time difference based on any timestamp will be negative
+   * and thus ignored until the app records initial time during [onCreate].
```

Comment:
This explains why the initial value was chosen, but not what this field actually is.

Reply:
done.